vis_server/protocols/FileLoader.py

```diff
@@ -24,7 +24,17 @@ def list_variables(self, file_name):
         out = {}
         for vname in reader.variables:
             var = reader.variables[vname]
+
+            # Get a displayable name for the variable
+            if hasattr(var, 'long_name'):
+                name = var.long_name
+            elif hasattr(var, 'title'):
+                name = var.title
+            elif hasattr(var, 'id'):
```

Comment:
This should just be `else:`, as I'm pretty sure every variable has an `id`. @dnadeau4 or @doutriaux1 may be more qualified to comment on that, though.